lambda/POST.js

```diff
@@ -1,97 +1,171 @@
-console.log('Loading function');
-//CloudWatch log - function successfully triggered notice
+console.log('Starting ActoKids Lambda function');
+//CloudWatch log - This log tells us the Lambda function successfully triggered.
 
-//Load required items and create DynamoDB doc client
 const AWS = require('aws-sdk');
 const docClient = new AWS.DynamoDB.DocumentClient({ region: 'us-west-2'});
 const uuid = require('uuid');
 
+//Variable needed to pass the timestamp for when the event is created in DynamoDB
 var now = (new Date()).toJSON();
 
-//create handler function
+//CloudWatch log - timestamp to verify it is accurate. CloudWatch already stores timestamps - can compare.
+console.log(`Lambda timestamp 'now' set as: ${now}`);
```

Comment:
I believe CloudWatch already handles date and time as you stated in the comments, so this may not be needed.  Are the CloudWatch timestamps buggy or inaccurate to the point we should have to verify?

Reply:
No, but the variable is necessary for the event object to be stored in DynamoDB and I figured it could be useful to check against the CloudWatch timestamp as well. That shows if some how it gets off. Also, Jon suggested the placement of setting the time may not be the best so testing with the log is helpful. You can see throughout the code/logs that the 'now' variable remains as it should, but the CloudWatch timestamp continues to increase as it should and so they diverge.

Reply:
Good point.  We should probably have a consistent logging pattern across our functions once we merge them together (the ones that will be merged that is) in the next sprint or later.